### DISCUSSION TITLE
Review button temporarily covered after marking files as viewed

### DISCUSSION THREAD

**Discussion by jmeridth** [community/community#63866] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.127.0 (107661963)
Device: iPhone15,2 (iOS 16.6)
</details>

When reviewing a PR after marking files as viewed the `Mark file as viewed` alert covers the `Review Changes` button. It is temporary but it happens enough that is now bothersome. I have to wait for the timeout to happen to access the review button. 

Example (button at bottom of app):

![260854748-59229c6f-0251-4a9e-b862-e2394c0a37dd](https://github.com/community/community/assets/35014/611a8a39-eab8-451a-8d8a-3c08707c7861)

**Comment by eliperkins** [comment, marked as answer]
Hey @jmeridth, thanks for this feedback!

That screenshot alone looks frustrating, I hear ya on this one. I've added this to a tracking issue for this bug in our internal repo. I'll work with our team to get it addressed.

Thanks for sharing your painpoints so that we can improve GitHub Mobile for ya!

**Comment by Jaber-Saed** [comment]
It's great that you've taken the initiative to track the bug and work with your team to address it. This kind of feedback loop is essential for improving software and user experiences.

**Comment by jmeridth** [comment]
@eliperkins any updates on this? Thank you.
